buildtest/cli/report.py

```diff
@@ -108,6 +109,7 @@ def __init__(
             oldest (bool, optional): Fetch oldest run for all tests discovered. This is specified via ``buildtest report --oldest``
             count (int, optional): Fetch limited number of rows get printed for all tests discovered. This is specified via ``buildtest report --count``
             pager (bool, optional): Enabling PAGING output for ``buildtest report``. This can be specified via ``buildtest report --pager``
+            format_detailed(bool,optional): Print a detailed summary of the test results. This can be specified via ``buildtest report --detailed``
```

Comment:
please put a space character between `,` and `optional`

Reply:
fixed

Reply:
No this is not fixed unless I am not seeing this properly please add space or share the commit where you did this